server/server.js

```diff
@@ -0,0 +1,62 @@
+import express from 'express';
+import mongoose from 'mongoose';
+import path from 'path';
+import webpack from 'webpack';
+import webpackMiddleware from 'webpack-dev-middleware';
+import webpackHotMiddleware from 'webpack-hot-middleware';
+import webpackConfig from '../webpack.config';
+
+import routes from './route';
+
+const app = express();
+const compiler = webpack(webpackConfig);
+
+  app.use(webpackMiddleware(compiler, {
+    hot: true,
+    publicpath: webpackConfig.output.publicPath,
+    stats: { colors: true },
+    noInfo: true
+  }));
+  app.use(webpackHotMiddleware(compiler));
+
+const staticMiddleware = express.static("client");
+
+app.use(staticMiddleware);
+
+// Set up mongoose connection
+const mongoDB = process.env.MONGODB_URI;
+mongoose.connect(mongoDB);
+mongoose.Promise = global.Promise;
+const db = mongoose.connection;
+db.on('error', console.error.bind(console, 'MongoDB connection error:'));
```

Comment:
Hi, @Orlayhemmy I clearly see you explicitly provided a  `logger` when connection to the database fails. It would also be helpful if you as well, explicitly provide a `logger mechanism` when connection to the database is successful. This can contribute to the connection debugging process of the mongo database, as the developer would be aware of both instances.
So, as with the `error logger`, you can do something like:
`db.once('open', () => { console.info({ message: 'Connected to the database' });
  });`, for the `success logger`.

Reply:
Oh...thanks very much for the initiative. I will implement the feedback and revert